app/scripts/lib/transaction/hooks/delegation-7702-publish.ts

```diff
@@ -106,37 +114,61 @@ export class Delegation7702PublishHook {
     const { delegationAddress, upgradeContractAddress } =
       atomicBatchChainSupport;
 
-    if (!selectedGasFeeToken || !gasFeeTokens?.length) {
+    const isGaslessBridge = transactionMeta.isGasFeeIncluded;
```

Comment:
Shouldn't this be called `isGaslessSwap`? Or we can preserve the name and call it `isGasFeeIncluded`

Reply:
makes sense, I'll go with isGaslessSwap here, to make it clear that this is not for the vanilla gas station flow.